orc8r/cloud/go/obsidian/swagger/handlers/handlers.go

```diff
@@ -33,3 +100,79 @@ func GetGenerateCombinedSpecHandler(yamlCommon string) echo.HandlerFunc {
 		return c.String(http.StatusOK, combined)
 	}
 }
+
+// getSpecHandler returns a routing handler which creates and
+// serves the raw YAML spec of a particular service.
+func getSpecHandler(yamlCommon string) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		service, ok, err := getServiceName(c.Param("service"))
+		if err != nil {
+			return obsidian.HttpError(err, http.StatusInternalServerError)
+		}
+		if !ok {
+			return obsidian.HttpError(errors.New("service not found"), http.StatusNotFound)
+		}
+
+		combined, err := swagger.GetServiceSpec(yamlCommon, service)
+		if err != nil {
+			return obsidian.HttpError(err, http.StatusInternalServerError)
+		}
+
+		return c.String(http.StatusOK, combined)
+	}
+}
+
+// getUIHandler returns a routing handler which serves
```

Comment:
Nit here and elsewhere: you're wrapping the doc comments too aggressively. Let them flow out toward the 79-character limit, then wrap

Reply:
Got it, will keep this in mind.